smoketest/scripts/cli/test_netns.py

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env python3
+#
+# Copyright (C) 2021 VyOS maintainers and contributors
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License version 2 or later as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+import re
+import os
+import json
+import unittest
+
+from netifaces import interfaces
+from base_vyostest_shim import VyOSUnitTestSHIM
+
+from vyos.configsession import ConfigSession
+from vyos.configsession import ConfigSessionError
+from vyos.ifconfig import Interface
+from vyos.ifconfig import Section
+from vyos.util import cmd
+
+base_path = ['netns']
+namespaces = ['mgmt', 'front', 'back']
+interfaces = ['dum10', 'dum12', 'dum50']
+
+class NETNSTest(VyOSUnitTestSHIM.TestCase):
```

Comment:
Why not derive from the already existing interface tests? You can derivce the individual testcases and assign interface to the netns prior of running the testcase itself.

Please also name this file: test_interfaces_netns.py to match the established pattern.

Reply:
Rename done.

Reply:
I renamed the test.
> Why not derive from the already existing interface tests?

This time `netns` option is allowed only for `dummy` interfaces